payment_alipay/README.md

```diff
@@ -0,0 +1,31 @@
+# payment_alipay
+alipay for odoo
+
+this module still in development, use it in production on your own risk
+
+any feedback should be thankful.
+
+if you want technical support, please contact the author and check the link http://item.taobao.com/item.htm?spm=a1z10.1-c.w4004-10020970078.16.EvfHHC&id=44624969034
```

Comment:
remove commercial information.
Contributors (individuals) are listed in the README while companies are inserted in __openerp__.py (authors).